lib/MetaCPAN/Document/Release.pm

```diff
@@ -1148,9 +1148,10 @@ sub modules {
     );
     return unless $ret->{hits}{total};
 
-    my @files = map { single_valued_arrayref_to_scalar($_) }
-        map +{ %{ $_->{fields} }, %{ $_->{_source} } },
-        @{ $ret->{hits}{hits} };
+    my @files = map {
+        single_valued_arrayref_to_scalar( $_->{fields} );
+        +{ %{ $_->{fields} }, %{ $_->{_source} } }
+    } @{ $ret->{hits}{hits} };
```

Comment:
that's what I tried first, but it produces an error:
Can't use string ("1") as a HASH ref ...

this code was a quick working compromise.